test/integration/buyers/accounts_controller_test.rb

```diff
@@ -294,6 +294,16 @@ def setup
       get admin_buyers_accounts_path(id: @buyer.id)
       assert_xpath('//tbody/tr/td[2]/a', @buyer.decorate.admin_user_display_name)
     end
+
+    test 'User with invalid data shows an flash error' do
+      post admin_buyers_accounts_path, account: {
+          org_name: 'My organization',
+          user: {
+            username: 'hello'
+          }
+      }
+      assert_equal 'Users invalid', flash[:error]
```

Comment:
The message should be `Users invalid`?
Maybe `User is invalid` is best?

Reply:
@damianpm you can also leave a suggestion so it can be committed easily ;)
```suggestion
      assert_equal 'Users is invalid', flash[:error]
```